test/cd.test.ts

```
@@ -0,0 +1,329 @@
+
+    it('should never write ANSI escape codes to stdout', async () => {
+        const { cdWorktreeHandler } = await import('../src/commands/cd.js');
+
+        mockStat.mockRejectedValueOnce(new Error('ENOENT'));
+        mockFindByBranch.mockResolvedValueOnce(mockWorktrees[0]);
+        mockStat.mockResolvedValueOnce({} as any);
+
+        await cdWorktreeHandler('main');
+
+        for (const call of stdoutSpy.mock.calls) {
+            expect(String(call[0])).not.toMatch(/\x1b\[/);
```

Biome (2.3.13)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)